net/relayd/src/opnsense/service/conf/actions.d/actions_relayd.conf

```diff
@@ -0,0 +1,29 @@
+[start]
+command:/usr/local/etc/rc.d/relayd start
+type:script
+message:starting relayd
+
+[stop]
+command:/usr/local/etc/rc.d/relayd stop
+type:script
+message:stopping relayd
+
+[status]
+command:/usr/local/etc/rc.d/relayd status || exit 0
+type:script_output
+message:get relayd status
+
+[restart]
+command:/usr/local/etc/rc.d/relayd restart
+type:script
+message:restarting relayd
+
+[reload]
```

Comment:
how useful is reload as opposed to stop+start? are you using it? if not consider removing it

Reply:
Reload doesn't stop the Relayd process. The process reads its configuration without service interruption.
Stop/Start will break the service. For redirects it's not so important but for relays.

Reply:
if you tested this that's ok to keep, most implementations don't do this (squid is the only one).

you want to implement a reload function helper for the MutableService so it works automatically:

https://github.com/opnsense/core/blob/master/src/opnsense/mvc/app/controllers/OPNsense/Proxy/Api/ServiceController.php#L46-L57

Reply:
Yes I want. :smile:
I'll do that.